templates/apidocs.html

```diff
@@ -1583,6 +1586,60 @@ <h3 class="panel-title">Example GET Request:</h3>
         </div>
       </div>
 
+      <h2 id="api-libraries">API Libraries</h3>
+      <p>
+        These libraries have been created by our community to help you use our
+        API in your next project.
+      </p>
+      <table class=table>
```

Comment:
Copy and pasted this. Probably should put some quotation marks around that class.